Gulpfile.js

```diff
@@ -19,6 +21,11 @@ const files = {
     js: './src',
     static: ['./www/**/*', './config.xml', './electron.js', '!./www/**/*.scss'],
     sass: './styles/*.scss',
+    androidApk: './platforms/android/build/outputs/apk',
+    unalignedApk: './platforms/android/build/outputs/apk/android-release-unaligned.apk',
+    alignedApk: './platforms/android/build/outputs/apk/android-release-aligned.apk',
+    zipAligned: '/Users/nuru/Library/Android/sdk/build-tools/23.0.3/zipalign', // Note the path to the zipalign on your pc
```

Comment:
@nuruddeensalihu `ANDROID_SDK_ROOT` is an environment variable that should be available after setting up an Android development environment. It points to the top-level SDK directory (so on your system, you could set it with `export ANDROID_SDK_ROOT=/Users/nuru/Library/Android/sdk`). Google don't document it very well, but it's used by several SDK tools - there are various stackoverflow.com posts describing how to set it on different OSes.

You might not need `gutil.env` for this - I think https://nodejs.org/api/process.html#process_process_env would work better.  The idea is that people should be able to do a build without needing to change `Gulpfile.js`.